src/ui/template.tsx

```diff
@@ -56,7 +65,9 @@ export function Template({
               <div class="content clearfix">
                 <div class="r">
                   <div class="c24">
-                    <div class="call">{children}</div>
+                    <div class="call" style="text-align: left;">
```

Comment:
Was centered intentionally. You can override that for your imprint / privacy policy